api/internal/git/cloner.go

```diff
@@ -44,6 +45,19 @@ func ClonerUsingGitExec(repoSpec *RepoSpec) error {
 			repoSpec.CloneSpec(), repoSpec.Dir.String())
 	}
 
+	cmd = exec.Command(
+		gitProgram,
+		"fetch",
+		"--depth=1",
+		"origin",
+		"FETCH_HEAD")
```

Comment:
Not here, use FETCH_HEAD in the checkout command in line 47

Reply:
So sorry, I understood where you wanted it to, just updated it wrong. My mistake. I'm fixing it now (and create a small test setup)